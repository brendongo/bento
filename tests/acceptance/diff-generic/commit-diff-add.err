Running Bento checks on changes since last commit...

⚠ No staged files to check.
◦ To check unstaged diffs, run $ bento check
◦ To check a specific path, run $ bento check PATH

✔ 0 findings in  s
◦ To show all findings in the project, run $ bento check --comparison=archive .
