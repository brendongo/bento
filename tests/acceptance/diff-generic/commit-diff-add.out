On branch master
Untracked files:
	.bento/
	.bentoignore
	bar.py

nothing added to commit but untracked files present
